Sources/ViewInspector/PopupPresenter.swift

```diff
@@ -102,6 +102,7 @@ public extension ItemPopupPresenter where Popup == ActionSheet {
 
 @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
 public extension ViewModifier where Self: BasePopupPresenter {
+    @MainActor
```

Comment:
These aren't necessary in the Xcode 16 beta, but the library doesn't compile in Xcode 15.4 without these extra annotations. They do not appear to cause any warnings when present in Xcode 16 beta.